README.md

```diff
@@ -1,97 +1,11 @@
-
-| Parameter | Description |
-|--------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `awsProduct` | Portworx Product Name, PX-ENTERPRISE or PX-ENTERPRISE-DR (Defaults to PX-ENTERPRISE) |
-| `clusterName` | Portworx Cluster Name |
-| `namespace` | Namespace in which to deploy portworx (Defaults to kube-system) |
-| `storage.usefileSystemDrive` | Should Portworx use an unmounted drive even with a filesystem ? |
-| `storage.usedrivesAndPartitions` | Should Portworx use the drives as well as partitions on the disk ? |
-| `storage.drives` | Semi-colon seperated list of drives to be used for storage (example: "/dev/sda;/dev/sdb"), to auto generate amazon disks use a list of drive specs (example: "type=gp2\,size=150";type=io1\,size=100\,iops=2000"). Make sure you escape the commas |
-| `storage.journalDevice` | Journal device for Portworx metadata |
-| `storage.maxStorageNodesPerZone` | Indicates the maximum number of storage nodes per zone. If this number is reached, and a new node is added to the zone, Portworx doesn’t provision drives for the new node. Instead, Portworx starts the node as a compute-only node. |
-| `network.dataInterface` | Name of the interface <ethX> |
-| `network.managementInterface` | Name of the interface <ethX> |
-| `secretType` | Secrets store to be used can be aws-kms/k8s/none defaults to: none |
-| `envVars` | semi-colon-separated list of environment variables that will be exported to portworx. (example: MYENV1=val1;MYENV2=val2) |
-| `serviceAcccount` | Name of the created service account with required IAM permissions |
-
-Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`.
-
-## Un-installing Portworx
-
-Edit the storage cluster.
-```bash
-kubectl -n <namespace> edit storagecluster <yourclustername>
-```
-
-Add the following yaml under `spec:`
-
-```bash
-deleteStrategy:
-    type: UninstallAndWipe
-```
-
-Save the spec and exit out.
-Then delete the storagecluster
-
-```bash
-kubectl -n <namespace> delete storagecluster <yourclustername>
-```
-
-Once all the portworx related pods are gone
-un-install/delete the `my-release` deployment:
-
-```
-helm delete my-release
-```
-This command removes all the Kubernetes components associated with the chart and deletes the release.
-
-## Documentation
-* [Portworx docs site](https://docs.portworx.com/scheduler/kubernetes/)
-* [Portworx interactive tutorials](https://docs.portworx.com/scheduler/kubernetes/px-k8s-interactive.html)
-
-## Support
-
-Please contact us at support@portworx.com with the generated log files.
-
-We are always available on Slack. Join us on [Slack](http://slack.portworx.com)
+## Helm Chart for PX-Central
+This installs PX-Backup plus PX-Lighthouse.
```

Comment:
Do we need to install lighthouse ?

Reply:
PX-Lighthouse is the UI components, I think because they are anyway getting installed, we may not need to explicitly mention it.